services/core-api/app/api/now_applications/models/activity_summary/sand_gravel_quarry_operation.py

```diff
@@ -52,14 +54,27 @@ class SandGravelQuarryOperation(ActivitySummaryBase):
     secure_access_plan = db.Column(db.String)
     dust_impact_plan = db.Column(db.String)
     visual_impact_plan = db.Column(db.String)
-
+    progressive_reclamation = db.Column(db.Boolean)
+    max_unreclaimed = db.Column(db.Numeric)
+    max_unreclaimed_unit_type_code = db.Column(
+        db.String, db.ForeignKey('unit_type.unit_type_code'))
     reclamation_backfill_detail = db.Column(db.String)
+    proposed_activity_description = db.Column(db.String)
+    work_year_info = db.Column(db.String)
 
     details = db.relationship(
         'SandGravelQuarryOperationDetail',
         secondary='activity_summary_detail_xref',
         load_on_pending=True)
 
+    # TODO replace with value from vFCBC
```

Comment:
Is this comment still valid?

Reply:
Yes, this value has not been sent to us. If "other" is selected, users can populate a input field. So i'm just updating the checkbox until we have that value
![Screen Shot 2021-06-09 at 12 13 51 PM](https://user-images.githubusercontent.com/25125247/121415039-29a6da00-c91c-11eb-8dea-5a1da723ca12.png)